custom_components/ferroamp/sensor.py

```diff
@@ -976,6 +976,42 @@ def update_state_from_events(self, events):
             return True
 
 
+class SinglePhaseFerroampSensor(KeyedFerroampSensor):
+    """Representation of a single phase Sensor"""
+
+    def __init__(self, name, entity_prefix, key, phase: str, unit: str | None, icon, device_id, device_name, interval, precision, config_id, **kwargs):
+        super().__init__(name, entity_prefix, key, unit, icon, device_id, device_name, interval, config_id, **kwargs)
+        if self._attr_state_class is None:
+            self._attr_state_class = SensorStateClass.MEASUREMENT
+        self._precision = precision
+        self._phase = phase
+        self._attr_unique_id = f"{self.device_id}-{self._state_key}-{self._phase}"
```

Comment:
-{self._phase.lower()} to keep ids in lowecase?

Reply:
Entity-id is slufified anyway so we don't need to handle that ourselves.
<img width="450" alt="image" src="https://github.com/henricm/ha-ferroamp/assets/328334/49ab0be0-2a6b-45de-8d52-b9d814c6d00a">